contracts/sol/IncrementalMerkleTree.sol

```diff
@@ -23,7 +23,7 @@ pragma solidity ^0.5.0;
 
 import { SnarkConstants } from "./SnarkConstants.sol";
 import { Hasher } from "./Hasher.sol";
-import { Ownable } from "@openzeppelin/contracts/ownership/Ownable.sol";
+import { Ownable } from "./openzeppelin-solidity//contracts/ownership/Ownable.sol";
```

Comment:
Can we import these contracts from node_modules instead? That would be easier to maintain.

```
import { Ownable } from "../node_modules/openzeppelin-solidity/contractscontracts/ownership/Ownable.sol"
```

Reply:
Yes! I hadn't figured out how to do that but I came across a fix and will push it.